moveit_ros/planning/CMakeLists.txt

```diff
@@ -79,7 +79,7 @@ add_subdirectory(constraint_sampler_manager_loader)
 add_subdirectory(planning_request_adapter_plugins)
 # add_subdirectory(planning_scene_monitor)
```

Comment:
So we're still blocked by `PlanningSceneMonitor`, but all other dependencies are available, correct?